enterprise/server-enterprise/src/test/java/org/neo4j/server/rest/security/EnterpriseAuthenticationDocIT.java

```diff
@@ -35,4 +47,31 @@ public void startServer( boolean authEnabled ) throws IOException
                 .build();
         server.start();
     }
+
+    @Test
+    public void shouldHavePredefinedRoles() throws Exception
+    {
+        // Given
+        startServerWithConfiguredUser();
+
+        // When
+        String method = "POST";
+        String path = "db/data/transaction/commit";
+        HTTP.RawPayload payload = HTTP.RawPayload.quotedJson(
+                "{'statements':[{'statement':'CALL dbms.listRoles()'}]}" );
+        HTTP.Response response = HTTP.withHeaders( HttpHeaders.AUTHORIZATION, challengeResponse( "neo4j", "secret" ) )
+                .request( method, server.baseUri().resolve( path ).toString(), payload );
+
+        // Then
+        assertThat(response.status(), equalTo(200));
+        ArrayNode errors = (ArrayNode) response.get("errors");
```

Comment:
Would be really nice to create a POJO and ask Jackson to deserialize response into it. Then we could assert on fields and not do just manual JSON parsing.